cmd/dashboard.go

```diff
@@ -134,6 +138,28 @@ var DashboardCmd = &cobra.Command{
 			}
 
 			<-portForward.GetStop()
+		} else {
```

Comment:
Please move code in else to cli\pkg\standalone\dashboard.go to align with other standalone implementations

Reply:
Good idea. Fixed.